codex-ui/src/vue/components/alert/Alert.types.ts

```diff
@@ -0,0 +1,83 @@
+import type { Ref } from 'vue';
+
+/**
+ * Various alert type
+ */
+export type Alertype = 'success' | 'error' | 'warning' | 'info' | 'default';
+
+/**
+ * position of alert, default position -> bottom-center
+ */
+export enum POSITION {
+
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  BOTTOM_CENTER = 'bottom-center'
+}
```

Comment:
lets get rid of this param as it is not used at the moment

Reply:
remove